docs/tutorial/writing-tests/test-case.rst

```diff
@@ -349,7 +389,9 @@ a packaging meta-data unit to our ``units/extended_tutorial.pxu`` file:
     Depends:
       jq
 
-If you now run the following command you will notice a validation error.
+If you have developed the other providers that Checkbox comes with, by running
```

Comment:
"developed or installed"

Reply:
Maybe different terminology might be better, e.g, "included those providers in your venv". 
Maybe there should be a small note or section somewhere saying "You have seen the `python3 manage.py <opts>` elsewhere. If you want to include tests from other providers, simply run this in the provider source dir such as `.../base/manage.py`"